src/Passes/manage

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import os
+import sys
+
+# Adding the site-packages directory to our Python path
+# in order to access the TasksCI module
+ROOT = os.path.dirname(__file__)
+sys.path.insert(0, os.path.join(ROOT, "site-packages"))
+
+# Loading the CLI tool and running it
+from TasksCI.cli import cli  # noqa: E402
+cli()
```

Comment:
Would be nice to have a comment with a web link here providing the introduction to what is going on here.

Reply:
This part of the code, imports the function `cli` from the [TasksCI](https://github.com/troelsfr/qsharp-compiler/tree/feature/llvm-pass-proposal/src/Passes/site-packages/TasksCI) which is a custom written Python package to deal with style checking and static analysis using `clang-format`, `clang-tidy` etc. 

I've added a comment on where to look for the details of the function `cli`. Let me know if you would like further changes.

Reply:
OK. I just didn't notice that TasksCI goes together with this PR (I thought it was some commonly known tool). 
Thanks.